certification/internal/shell/validate_operator_bundle.go

```diff
@@ -1,42 +1,59 @@
 package shell
 
 import (
-	"os/exec"
-	"strings"
-
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
 	log "github.com/sirupsen/logrus"
 )
 
-type ValidateOperatorBundlePolicy struct {
+type ValidateOperatorBundleCheck struct {
```

Comment:
Nit: Same here re: check naming

Reply:
Isn't this the correct name? I am changing it to Check from Policy.

Reply:
By naming here, I meant the adjustment from ValidateOperatorBundle to something more prescriptive. That said, I'll mark this resolved because changing this in the previous comment would require it to fix this variable reference as well. There's no reason for me to have commented twice here. My apologies!